tests/testthat/test-score.R

```diff
@@ -203,7 +160,7 @@ test_that("score_quantile correctly handles separate results = FALSE", {
   df <- example_quantile[model == "EuroCOVIDhub-ensemble" &
                            target_type == "Cases" & location == "DE"] %>%
     as_forecast()
-  eval <- score(df[!is.na(predicted)], separate_results = FALSE)
+  eval <- score(df[!is.na(predicted)])
```

Comment:
maybe keeping that test (and changing `wis` to `customise_metric(wis, separate_results = FALSE)`) would be good. But I guess we probably have that covered somewhere else.

Reply:
I added that to my PR with suggested changes